CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.6
+
+<!-- generated comparing v5.14.5..master -->
+
+_Aug 23, 2023_
+
+A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Added [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak
```

Comment:
It would be awesome to have a paragraph or two on this. For example, to share our plan when it comes to Popover vs. Popper vs. Popup. But also, when to use each, and lastly, why is there a new one. I personally didn’t understand this. Why the library it uses under the hood feels like an implementation details.

If I would try to rationalize it, it seems to be: since we are moving Popper to no longer use popper.js, we do no longer need to call this component Popper. We are renaming it to Popup, which keeping the API almost identical for easy migration.